docs/use.md

```diff
@@ -0,0 +1,106 @@
+# User guide
+
+These sections describe how to control the major functionality of this tool.
+
+## Generate a markdown changelog
+
```

Comment:
Is the expectation that a user should be able to follow this from top to bottom the first time they use this tool? If so I think mentioning the need for a GitHub token first would be helpful since it's mandatory
https://github.com/executablebooks/github-activity/blob/1ce5486df300e4b6be88753fccb891f09790f9b2/github_activity/github_activity.py#L159-L168